### DISCUSSION TITLE
Possibility to add a CardDAV to paperless to auto detect corespondent

### DISCUSSION THREAD

**Discussion by Mannshoch** [jonaswinkler/paperless-ng#1399] [Feature Requests]
It  would be interesting if Papaerless-NG could use CardDAV to detect addresses on PDF and are able to auto assign PDF to specific corespondents.

I would propose to search for Addresses and Pre/Lastname.

**Comment by thomassaffer** [comment]
Interesting feature, also the other way round, creatingCardDAV contact from correspondents...

**Comment by mtettke123** [comment]
Wow, great idea. Although I would love to have it only one way, don't want paperless-ng to edit my contacts...

**Reply by thomassaffer** [reply]
One way would be totally great!

**Comment by MAngel666** [comment]
+1
